src/lobby/room-instance.js

```diff
@@ -98,6 +98,7 @@ class LobbyRoomInstance extends React.Component {
           {inst.players.map(this._createSeat).join(', ')}
         </td>
         <td key={'cell-buttons-' + inst.gameID}>
+          {/* TODO show leave button even if game is ready to play */}
```

Comment:
@nicolodavis Could you please add this TODO to #354 improvement as well?

Reply:
Done.

Reply:
Thanks!